codenames-front-end/src/components/game/running/CNGameWord/StyledCNGameWord.ts

```diff
@@ -0,0 +1,47 @@
+import {Color} from "../../../../models/CodeNames/Color";
+import {CNGameWord} from "./CNGameWord";
+import styled from "styled-components";
+
+const colorStylesMap = new Map<Color, string>([
+    [Color.YELLOW, "var(--words-yellow-color)"],
+    [Color.BLUE, "var(--words-blue-color)"],
+    [Color.BLACK, "var(--words-black-color)"],
+    [Color.WHITE, "var(--words-white-color)"]
+])
+
+type StyleProps = {
+    wordColor: Color;
+    className?: string
+}
+
+export const StyledCNGameWord = styled(CNGameWord)<StyleProps>`
+  user-select: none;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  
+  margin: 10px;
+
+  width: var(--words-width);
+  height: 90px;
+
+  color: ${props => props.wordColor === Color.BLACK ? "#ffffff" : "#1E1E1E"};
+  background-color: ${props => props.wordColor ? colorStylesMap.get(props.wordColor) : "var(--words-default-color)"};
+  
+  font-style: normal;
+  font-weight: 600;
+  font-size: 25px;
```

Comment:
These pixel-perfect measurements seem very specific and maybe should be switched to smth like em/rem
Have you tested your app on different screen resolutions?

Reply:
Yes, I tested it, everything works fine, but I don't make pages for phones and tablets because it's a lot of work and I wouldn't want to do it for educational purposes

Reply:
**This is a comment to the post about directory structure**

I've reworked it a bit, but I don't quite understand what else needs to be fixed